### PR TITLE
fix: release note draft

### PR DESCRIPTION
**Pull request by yuwenma**
### Change description

Fixes #

### Tests you have done

- [ ] Run `make ready-pr` to ensure this PR is ready for review.
- [ ] Perform necessary E2E testing for changed resources.

### CODE REVIEW

File: docs/releasenotes/release-1.125.md [hunk 1]
```diff
@@ -2,42 +2,61 @@
...
-  * This release adds this feature in the cluster mode in `ControllerReconciler` object. This [example](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/operator/config/samples/controller_reconciler_customization_sample.yaml) shows how to set up the configuration.
+  * In v1.119, we added [rate-limit control in namespace mode](https://cloud.google.com/config-connector/docs/how-to/customize-controller-manager-rate-limit). Users can configure `NamespacedControllerReconciler` object (Alpha) to set the rate-limit for the reconciling requests to the kube-apiserver for their Config Connector resources. 
+  * In this release, we add this feature for cluster mode. User can configure `ControllerReconciler` object (Alpha) to set the rate-limit for all their cnrm manager controllers in the cluster. This [example](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/operator/config/samples/controller_reconciler_customization_sample.yaml) shows how to set up the configuration.
```
Comment (cheftako) [inline docs/releasenotes/release-1.125.md:56]: Users can

File: docs/releasenotes/release-1.125.md [hunk 2]
```diff
@@ -2,42 +2,61 @@
...
   * Added `spec.networkConfig.enableOutboundPublicIp` field.
   * Added `status.outboundPublicIpAddresses` field.
 
-## Modified Beta Reconciliation
+## Reconciliation Improvements
 
 We migrated the following resources from the Terraform-based or DCL-based controller to the new Direct Controller to enhance the reliability and performance. The resource CRD is unchanged.
 
-* `SQLInstance`
+* [`SQLInstance`](https://github.com/GoogleCloudPlatform/k8s-config-connector/tree/master/pkg/test/resourcefixture/testdata/basic/sql/v1beta1/sqlinstance)
```
Comment (cheftako) [inline docs/releasenotes/release-1.125.md:39]: @jasonvigil check my recollection. We have added support for running SQLInstance via Direct Controller but not actually migrated to it?

Comment (jasonvigil) [reply docs/releasenotes/release-1.125.md:39]: That's fair. Maybe the wording should say we "enabled" the direct controller, instead of "migrated"

Comment (yuwenma) [reply docs/releasenotes/release-1.125.md:39]: Thanks! I updated the release note to reflect the discussion

**Comment by yuwenma** [comment]
close since the release note has been udpated

**Pull request closed without merging**